src/components/organisms/UiHorizontalPaging/UiHorizontalPaging.vue

```diff
@@ -226,6 +226,7 @@ const menuItems = computed<MenuItemAttrsProps[]>(() => itemsAsArray.value.map((i
         item,
       ];
     },
+    tabindex: isActive.value ? '-1' : null,
```

Comment:
@AlanLes I have one TS error; Can you help me and know how to fix it?

Reply:
Hello, sure what's the error? :)

Reply:
I'm afraid I can't fetch your remote locally on my machine; I have no access rights and can't investigate it ;/ 
Maybe we'll huddle later on and investigate it together? :)